baseframe/static/js/baseframe-material.js

```diff
@@ -249,6 +249,26 @@ window.Baseframe.Forms = {
         }
       }
     })
+  },
+  showValidationErrors: function(formId, errors) {
+    Object.keys(errors).forEach(function(fieldName) {
+      if (Array.isArray(errors[fieldName])) {
+        var form = document.getElementById(formId);
+        var fieldWrapper = form.querySelector("#field-" + fieldName);
```

Comment:
Why `querySelector` and not `getElementById`?

Reply:
getElementById is not available here